server/routes/visitors.test.ts

```diff
@@ -1,15 +1,19 @@
 import type { Express } from 'express'
 import request from 'supertest'
 import { SessionData } from 'express-session'
-import { VisitorListItem } from '../@types/bapv'
+import * as cheerio from 'cheerio'
+import { VisitorListItem, VisitSessionData } from '../@types/bapv'
 import PrisonerVisitorsService from '../services/prisonerVisitorsService'
-import appWithAllRoutes from './testutils/appSetup'
+import { appWithAllRoutes, flashProvider } from './testutils/appSetup'
 
 let app: Express
+let sessionApp: Express
 let prisonerVisitorsService: PrisonerVisitorsService
-let systemToken
+let systemToken: (user: string) => Promise<string>
```

Comment:
Actually occurred to me while doing my test updates that this can just be set to a `const` as the value it is on line 30 as it never changes.

Reply:
👍 will do this as part of a subsequent PR